CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 ### New features:
  * `Address.toPayable`: added a helper to convert between address types without having to resort to low-level casting. ([#1773](https://github.com/OpenZeppelin/openzeppelin-solidity/pull/1773))
  * Facilities to make metatransaction-enabled contracts through the Gas Station Network. ([#1844](https://github.com/OpenZeppelin/openzeppelin-contracts/pull/1844))
+ * `Address.sendEther`: added a replacement to Solidity's `transfer`, removing the fixed gas stipend. ([#1961](https://github.com/OpenZeppelin/openzeppelin-solidity/pull/1961))
```

Comment:
Isn't the "Ether" naming too network coupled? I mean this aiming to be standard solidity contracts that should work with any compible EVM. (RSK or Cardano just to name a few)

Reply:
Hm, this is a good point. What would you suggest? `sendFunds`?